src/Common/src/System/Net/SafeCloseSocket.Unix.cs

```diff
@@ -224,6 +224,14 @@ private void InnerReleaseHandle()
             }
         }
 
+        private void TryWakeup(InnerSafeCloseSocket innerSocket)
```

Comment:
nit: I would rename this to remove the 'Try' prefix since methods with that prefix have an expected pattern that doesn't match this.